listenbrainz/db/dump.py

```diff
@@ -671,7 +671,7 @@ def get_dump_entries():
               ORDER BY created DESC
             """))
 
-        return [dict(row) for row in result]
```

Comment:
try `result.mappings().all()` or `[dict(row) for row in result.mappings()]`

Reply:
Done